pkg/apis/core/validation/validation.go

```diff
@@ -1770,20 +1770,30 @@ func ValidatePersistentVolume(pv *core.PersistentVolume) field.ErrorList {
 // ValidatePersistentVolumeUpdate tests to see if the update is legal for an end user to make.
 // newPv is updated with fields that cannot be changed.
 func ValidatePersistentVolumeUpdate(newPv, oldPv *core.PersistentVolume) field.ErrorList {
+	newPvClone := newPv.DeepCopy()
+	oldPvClone := oldPv.DeepCopy()
+
 	allErrs := field.ErrorList{}
-	allErrs = ValidatePersistentVolume(newPv)
+	allErrs = ValidatePersistentVolume(newPvClone)
+
+	if utilfeature.DefaultFeatureGate.Enabled(features.ExpandCSIVolumes) {
```

Comment:
no need to check the feature gate here... if the feature's disabled and oldPv doesn't have the field set, the strategy will clear it from newPv as well before we get here